### DISCUSSION TITLE
Release 2.0

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/logicanalyzer#18] [Announcements]
This release contains the new multiplatform app and corrects problems with the command line app under Raspberry.

<hr /><em>This discussion was created from the release <a href='https://github.com/gusmanb/logicanalyzer/releases/tag/v2.0.0.0'>Release 2.0</a>.</em>

**Comment by terenceyhj** [comment]
verygood, Hi。Whether to continue release[-win-x86...].Tks!
